node/adapters/newsfeed.go

```diff
@@ -0,0 +1,22 @@
+package adapters
+
+import (
+	"github.com/status-im/status-go/protocol"
+)
+
+type NewsFeedActivityCenterAdapter struct {
```

Comment:
```suggestion
type NewsFeedActivityCenter struct {
```

usage: `adapters.NewsFeedActivityCenter`